apps/site-projects/src/components/modules/Resources/ProjectResources/TeamLinks/StyledResourceCard.js

```diff
@@ -1,11 +1,68 @@
-import styled from "styled-components";
-
-export const ResourceCardContainer = styled.div`
-    min-width: 14rem;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    border: 1px solid black;
-    border-radius: .5rem;
-    padding: 2rem 0rem;
-`;
\ No newline at end of file
+import styled from "styled-components";
+
+
+export const ResourcesContainer = styled.fieldset`
+    display:flex;
+    flex-direction: row;
+    gap:2rem;
+    background: white;
+    border: 3px solid #528cb0 ;
+    border-radius: 1rem;
+    padding: 2rem 3rem;
+    margin: 4rem 0;
+    padding-bottom: 2rem;
+    flex-wrap: wrap;
+    justify-content: center;
```

Comment:
also here to make the elements wrap just like the design and not centered with flex should maybe look something like this : 
```
export const ResourcesContainer = styled.fieldset`
    display:grid;
    grid-template-columns: repeat(auto-fit,minmax(15rem,16rem));
    justify-content: center;
    gap:2rem;
    background: white;
    border: 3px solid #528cb0 ;
    border-radius: 1rem;
    padding: 2rem 3rem;
    margin: auto 0;
    padding-bottom: 2rem;
`;
```
i made it a grid so the elements are centered but wrap elements just like the design, you tweak and play around with it until it feels right


![68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3634393231313832623861323337306464626630663962322f37376466613937352d646130662d346666392d393233642d353864616164343662656264](https://github.com/dev-launchers/dev-launchers-platform/assets/60440238/32a6008a-7af7-47af-bdf1-6738d992c02b)

Reply:
thank you, i've done that.